test/ValueProxy.spec.js

```diff
@@ -152,4 +153,54 @@ describe('ValueProxy', function () {
         });
     });
 
-});
\ No newline at end of file
+    describe('generations', function() {
+        before(() => Planner.enableLock());
```

Comment:
This can affect other test suites (due to static nature of Planner). You'd better use `new Planner` here